examples/scripts/data_io/dataframe/validate_dataframe.py

```diff
@@ -0,0 +1,40 @@
+from tamr_toolbox.models.validation_check import ValidationCheck
+import pandas as pd
+from tamr_toolbox.data_io import dataframe
```

Comment:
Our example scripts should a brief docstring at the top.

Also, it's a nice practice to organize imports.  One way is separating standard library imports (you don't have any of these), third-party imports (`pandas`), and local package imports (`tamr_toolbox`):
```suggestion
"""Example script for validating the contents of a DataFrame"""
import pandas as pd

from tamr_toolbox.data_io import dataframe
from tamr_toolbox.models.validation_check import ValidationCheck
```